src/main/java/hockeyapp/HockeyappRecorder.java

```diff
@@ -169,10 +190,21 @@ private HttpClient createPreconfiguredHttpClient() {
             httpclient.getParams().setParameter(ConnRoutePNames.DEFAULT_PROXY, proxy);
         }
 
+        /// Logging output ////////////////
```

Comment:
The white space formatting here is inconsistent with rest of the file / project. Please clean it up.